pages/linux/umask.md

```diff
@@ -0,0 +1,15 @@
+# umask
+
+> Display file creation mask or set file creation mask to octal value.
+
+- Display current value as octal:
+
+`umask`
+
+- Display current value symbolically:
+
+`umask -S`
+
+- Turn off permission bits with octal value 'nnnn':
```

Comment:
I'm still confused tbh, especially with the "turn off" part. Here's what Wikipedia says, which seems straightforward to me:

`umask 0077        # set the mask to 0077`